apps/drupal-default/particle_theme/templates/navigation/menu--utc-sidemenu.html.twig

```diff
@@ -40,7 +40,7 @@
           'menu-item-sidemenu',
           item.is_expanded ? 'menu-item--expanded',
           item.is_collapsed ? 'menu-item--collapsed',
-          item.in_active_trail ? 'menu-item--active-trail',
+          item.in_active_trail ? 'menu-item--active-trail show-sub',
```

Comment:
Good solution, this makes sense and is not hacky. However, does this do anything for the "+" click to open any submenu?